### PR TITLE
Create ui-c8y-1018-503-47-3-d-rotation-model-shows-the-actual-model-5565.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57985](https://cumulocity.atlassian.net/browse/MTM-57985)
Original PR: [5565](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5565)

### CODE REVIEW

**Comment by janhommes** [comment]
this is our automation test. This PR is for y2024. This needs to go to a different branch, or?

**Comment by BeateRixen** [comment]
> this is our automation test. This PR is for y2024. This needs to go to a different branch, or?

Yes. Must go to release/y2024.

File: content/change-logs/ui-c8y-1018-503-47-3-d-rotation-model-shows-the-actual-model-5565.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57985
+version: 1018.503.47
+---
+An issue has been addressed that caused the 3D rotation widget to not show the actual model.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1018-503-47-3-d-rotation-model-shows-the-actual-model-5565.md:17]:
```suggestion
A bug has been fixed that prevented the "Rotation" widget from displaying the actual model.
```

File: content/change-logs/ui-c8y-1018-503-47-3-d-rotation-model-shows-the-actual-model-5565.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: 3D rotation model shows the actual model (#5565)
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1018-503-47-3-d-rotation-model-shows-the-actual-model-5565.md:3]:
```suggestion
title: Rotation widget shows the actual model (#5565)
```

**Comment by janhommes** [comment]
> > this is our automation test. This PR is for y2024. This needs to go to a different branch, or?
> 
> Yes. Must go to release/y2024.

then we would need to rebase it to this branch for now... will look into the automation to do this automatically.

**Comment by janhommes** [comment]
wrong branch.

**Pull request closed without merging**